modules/fx/arrow-fx/src/main/kotlin/arrow/fx/typeclasses/Async.kt

```diff
@@ -222,11 +221,11 @@ interface Async<F, E> : MonadDefer<F, E> {
    * }
    * ```
    */
-  fun <A> effect(ctx: CoroutineContext, f: suspend () -> A): Kind<F, A> =
-    effect(ctx, ::identity, f)
+//  fun <A> effect(ctx: CoroutineContext, f: suspend () -> A): Kind<F, A> =
+//    effect(ctx, ::identity, f)
```

Comment:
mapError should be on `ApplicativeError`, if it isn't there already.

Reply:
this is an old commit, for now i simply limited effect to `Async<F, Throwable>` only, but there's no mapError anywhere except in IO itself